app/src/main/kotlin/com/wire/android/ui/WireActivityDialogs.kt

```diff
@@ -363,6 +390,142 @@ fun NewClientDialog(
     }
 }
 
+@Composable
+fun CallFeedbackDialog(
+    sheetState: WireModalSheetState<Unit>,
+    onRated: (Int, Boolean) -> Unit,
+    onSkipClicked: (Boolean) -> Unit
+) {
+    val doNotAskAgain = remember {
+        mutableStateOf(false)
+    }
+    WireModalSheetLayout(
+        sheetState = sheetState,
+        sheetContent = {
+            Column(modifier = Modifier.padding(all = dimensions().spacing24x)) {
+                Text(
+                    modifier = Modifier.align(Alignment.CenterHorizontally),
+                    text = stringResource(R.string.call_feedback_dialog_title),
+                    style = MaterialTheme.wireTypography.title01
+                )
+
+                Spacer(modifier = Modifier.height(dimensions().spacing16x))
+
+                Text(
+                    modifier = Modifier
+                        .align(Alignment.CenterHorizontally)
+                        .padding(horizontal = dimensions().spacing16x),
+                    text = stringResource(R.string.call_feedback_dialog_message),
+                    style = MaterialTheme.wireTypography.body01
+                )
+
+                Spacer(modifier = Modifier.height(dimensions().spacing32x))
+
+                Row(modifier = Modifier.fillMaxWidth()) {
+                    Text(
+                        stringResource(R.string.call_feedback_dialog_label_bad),
+                        style = MaterialTheme.wireTypography.label01,
+                        modifier = Modifier.weight(1F).padding(horizontal = dimensions().spacing8x),
+                        textAlign = TextAlign.Start
+                    )
+                    Text(
+                        stringResource(R.string.call_feedback_dialog_label_fair),
+                        style = MaterialTheme.wireTypography.label01,
+                        modifier = Modifier.weight(1F),
+                        textAlign = TextAlign.Center
+                    )
+                    Text(
+                        stringResource(R.string.call_feedback_dialog_label_excellent),
+                        style = MaterialTheme.wireTypography.label01,
+                        modifier = Modifier.weight(1F).padding(horizontal = dimensions().spacing8x),
+                        textAlign = TextAlign.End
+                    )
+                }
+
+                Spacer(modifier = Modifier.height(dimensions().spacing8x))
+
+                Row(modifier = Modifier.fillMaxWidth()) {
+
+                    WireSecondaryButton(
+                        onClick = { sheetState.hide { onRated(1, doNotAskAgain.value) } },
+                        state = WireButtonState.Default,
+                        text = "1",
```

Comment:
I think not every language uses arabic numbers, should we extract them to strings.xml?

Reply:
We can use `android.icu.text.NumberFormat` in that case.
I would suggest to also use loop to not have too much duplicated code and make use of `horizontalArrangement` to not add that many spacers 😄 
```
    Row(
        horizontalArrangement = Arrangement.SpaceBetween,
        modifier = Modifier.fillMaxWidth()
    ) {
        val numberFormat = NumberFormat.getNumberInstance()
        for (i in 1 .. 5) {
            WireSecondaryButton(
                onClick = { sheetState.hide { onRated(i, doNotAskAgain.value) } },
                state = WireButtonState.Default,
                text = numberFormat.format(i),
                modifier = Modifier.size(dimensions().spacing56x),
                shape = CircleShape
            )
        }
    }
```

Reply:
damn, forgot about it, thanks!!!